sorc/chgres_cube.fd/write_data.F90

```diff
@@ -2580,6 +2606,17 @@ subroutine write_fv3_sfc_data_netcdf(localpet)
      error = nf90_put_var( ncid, id_snoalb, dum2d, start=(/1,1,1/), count=(/i_target_out,j_target_out,1/))
      call netcdf_err(error, 'WRITING MAX SNOW ALBEDO RECORD' )
    endif
+   
```

Comment:
If LAI is not being processed, what gets written to the file?  Missing values?  Does that record need to be written?

Reply:
I've now added in logic that will only create and write to these output fields if lai_from_climo is true.